src/test/java/ai/elimu/web/content/word/WordCreateControllerTest.java

```diff
@@ -54,4 +58,56 @@ public void testHandleSubmit_emptyText() throws Exception {
 //        assertEquals(HttpStatus.BAD_REQUEST.value(), mvcResult.getResponse().getStatus());
 //        assertEquals("content/word/create", mvcResult.getModelAndView().getViewName());
     }
+
+    @Test
+    public void testValidateDigitsInWord() throws Exception {
+        assertTrue(containGivenErrorCode(getBindingResult(getRequestWithSpecificText("10")).getAllErrors(), "WordNumbers"));
```

Comment:
```suggestion
        assertTrue(containGivenErrorCode(getBindingResult(getRequestWithSpecificText("10")).getAllErrors(), "WordNumbers"));
        assertFalse(containGivenErrorCode(getBindingResult(getRequestWithSpecificText("10th")).getAllErrors(), "WordNumbers"));
```

Reply:
"10th" is accepted as word but I can't add this test because occuring error durign load EnvironmentContextLoaderListener

> WordCreateController.java

`            String contentUrl = "https://" + EnvironmentContextLoaderListener.PROPERTIES.getProperty("content.language").toLowerCase() + ".elimu.ai/content/word/edit/" + word.getId();
`
EnvironmentContextLoaderListener.PROPERTIES return null during test.

Reply:
@tomaszsmy Thank you for sharing this. I see that many of the controllers are using `EnvironmentContextLoaderListener.PROPERTIES.getProperty`, so added a fix for this here: https://github.com/elimu-ai/webapp/pull/1608

After merging that pull request to `main`, and then adding the changes to your branch, it should work to successfully store "10th" in your test 🙂